tests/s3/copy_object/test_copy_obj_simultaneous_del.py

```diff
@@ -219,3 +276,51 @@ def test_45525(self):
                                                         is_mpu=True)
         LOGGER.info("ENDED: Test Delete source object when copy multipart object is in "
                     "progress in same bucket")
+
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_object_copy
+    @pytest.mark.tags("TEST-45526")
+    @CTFailOn(error_handler)
+    def test_45526(self):
+        """Test Delete destination object when copy multipart object is in progress
+        in same bucket"""
+        LOGGER.info("STARTED: Test Delete destination object when copy multipart object is in "
+                    "progress in same bucket")
+        self.create_put_parallel_copy_and_delete_object((self.srcbuck, "src-obj", self.srcbuck,
+                                                         "dest-obj", "NoSuchKey", 5,
```

Comment:
Let's update the error messages string literals in this file to use the constants in the commons.error_messages module

Reply:
done